manager/integration/Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM python:2.7.14-stretch
 
+ARG KUBECTL_VERSION=v1.15.0
+
 RUN apt-get update && \
-    apt-get install -y vim-tiny nfs-common
+    apt-get install -y vim-tiny nfs-common && \
+    wget -q https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/linux/amd64/kubectl && \
```

Comment:
This doesn't belong to the `apt-get` lines. Add a new step instead.

Reply:
fixed